packages/react/src/components/form/Search/Search.stories.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import type { Meta, StoryObj, StoryFn } from '@storybook/react';
+import { MagnifyingGlassIcon } from '@navikt/aksel-icons';
+
+import { Button, Paragraph } from '../..';
+
+import { Search } from '.';
+
+type Story = StoryObj<typeof Search>;
+
+export default {
+  title: 'Felles/Search',
+  component: Search,
+} as Meta;
+
+export const Preview: Story = {
+  args: {
+    label: 'Label',
+    disabled: false,
+    size: 'medium',
+    error: '',
+    placeholder: '',
+    variant: 'simple',
+  },
+};
+
+export const FullWidth: Story = {
+  args: {
+    label: 'Label',
+    variant: 'primary',
+  },
+  parameters: {
+    layout: 'padded',
+  },
+};
+
+export const Controlled: StoryFn<typeof Search> = () => {
+  const [value, setValue] = useState<string>();
+  return (
+    <>
+      <Paragraph>Du har skrevet inn: {value}</Paragraph>
+      <div
+        style={{
+          display: 'flex',
+          flexDirection: 'column',
+          marginTop: 'var(--fds-spacing-2)',
+          gap: 'var(--fds-spacing-2)',
+        }}
+      >
+        <Search
+          label='Kontroller meg!'
+          value={value}
+          onChange={(e) => setValue(e.target.value)}
+        />
+        <Button onClick={() => setValue('Pizza')}>Jeg vil ha Pizza</Button>
+      </div>
+    </>
+  );
+};
+
```

Comment:
I am not able to use the clear button on this story

Reply:
This should be fixed, story had a missing handler for `onClear` which needs to be handled outside the component when its controlled :)